### ISSUE TITLE
Bogus error message when checking a specification with temporal existential quantification

### ISSUE THREAD

**Issue by lemmy** [tlaplus/tlaplus#304]
As [reported on the Google group](https://groups.google.com/forum/#!msg/tlaplus/bcP4VLhR9zY/lxhhtPdPAgAJ):

"[...] the specification from module BoundedFIFO is defined as

\EE q : Inner(q)!Init /\ [][BNext(q)]_<<in,out,q>>

and TLC does not support \EE, i.e. quantification over state variables. (I agree that the error message is not ideal.)"

```
TLC threw an unexpected exception.
This was probably caused by an error in the spec or model.
See the User Output or TLC Console for clues to what happened.
The exception was a java.lang.RuntimeException
:
The configuration file did not specify the initial state predicate.
Can also be caused by trying to run TLC on a specification from
a module imported with a parameterized INSTANCE statement.
```

**Comment by quaeler** [comment]
Addressed with 6569e88

**Comment by lemmy** [comment]
Maybe take temporal *universal* quantification into account too?

**Comment by quaeler** [comment]
Ah yes, i see that this produces an error, with incorrect verbiage, as well - will do.

**Comment by quaeler** [comment]
Addressed with 3932995 & 0958a59
